pyme-depends/meta.yaml

```diff
@@ -127,16 +127,16 @@ requirements:
   # in requirements" below).
   run:
     - python {{ python }}
-    - numpy >= 1.4
+    - numpy >=1.4

```

Comment:
Change this to `numpy >=1.14`

Reply:
whoops, did that on the server I'd ssh'd but not local. Thanks @zacsimile